### ISSUE TITLE
Update error message

### ISSUE THREAD

**Issue by KrauseFx** [danger/danger#184]
> Updating the Danger gem might fix the issue.

This message should only be shown if the user actually uses an outdated version of the gem.

**Comment by hanneskaeufler** [comment]
How would we know if the installed danger is the current version?

```
is_outdated = `bundle outdated danger`.include?("* danger")
```

comes to mind but ist that worth doing every time an error is raised? Danger might not even be installed through bundler all the time but as a gem directly.

Maybe remove the "Updating the Danger gem ..." text and not bother with it?

**Comment by marcelofabri** [comment]
We can use RubyGems API: https://rubygems.org/api/v1/versions/danger.json

**Comment by hanneskaeufler** [comment]
Cool, did not know that. Seems like a lot of work + code for networking / networking error handling etc. for something pretty minor?

**Comment by orta** [comment]
Aye, the concept comes from CocoaPods, where there is a guarantee of a local copy of the latest version as a string anytime an action is called. So it's easy there.

**Comment by JuanitoFatas** [comment]
I'm working on this 😄

**Comment by JuanitoFatas** [comment]
Implemented in #503.
